mix.exs

```diff
@@ -65,4 +75,24 @@ defmodule Wallaby.Mixfile do
       main: "readme",
     ]
   end
+
+  defp test_paths(driver) when driver in @drivers, do: ["integration_test/#{driver}"]
+  defp test_paths(_), do: ["test"]
+
+  defp test_drivers(args) do
+    for driver <- @drivers, do: run_integration_test(driver, args)
+  end
+
+  defp run_integration_test(driver, args) do
+    args = if IO.ANSI.enabled?, do: ["--color"|args], else: ["--no-color"|args]
+
+    IO.puts "==> Running tests for WALLABY_DRIVER=#{driver} mix test"
+    {_, res} = System.cmd "mix", ["test"|args],
+                          into: IO.binstream(:stdio, :line),
+                          env: [{"WALLABY_DRIVER", driver}]
+
+    if res > 0 do
+      System.at_exit(fn _ -> exit({:shutdown, 1}) end)
+    end
```

Comment:
This all seems really manual :scream: 

I've never done anything like it and I guess you were inspired by ecto and I guess they can be trusted with that, I just wished/thought there would be a more automated/less involved way to do this (not needing to send own System mix commands, but just... invoking another mix task as in "this should also run" a bit more like rake from the ruby world)